apps/idos-data-dashboard/src/routes/dashboard/wallets/components/add-wallet.tsx

```diff
@@ -57,14 +82,16 @@ export const AddWallet = ({ isOpen, onClose }: AddWalletProps) => {
     addWallet.mutate(
       { address },
       {
-        async onSuccess() {
+        async onSuccess(wallet) {
+          const cache = queryClient.getQueryData<idOSWallet[]>(["wallets"]) ?? [];
+          const updated = cache.map((_wallet) => (!_wallet.id ? { ...wallet } : _wallet));
```

Comment:
You're using the var, so it shouldn't be prefixed with `_`. Unsure what would be a good name, but I'd be happier with merely `wallet_` instead.